packages/overlay/src/ActiveOverlay.ts

```diff
@@ -488,8 +488,9 @@ export class ActiveOverlay extends SpectrumElement {
     private stealOverlayContentPromise = Promise.resolve();
     private stealOverlayContentResolver!: () => void;
 
-    protected async _getUpdateComplete(): Promise<void> {
-        await super._getUpdateComplete();
+    protected async getUpdateComplete(): Promise<boolean> {
+        await super.getUpdateComplete();
         await this.stealOverlayContentPromise;
+        return true;
```

Comment:
@Westbrook All these signature changes for `_getUpdateComplete` would be a breaking change.

Reply:
Yep, that's why I updated the commit to `feat`. While pre-1.0, semver registers any 0.x.0 change as _breaking_. Our release tooling with Lerna and Conventional Commits, however, register any reference to "breaking" as a x.0.0 change, which I'm not sure the work requires and/or we're fully ready for.

I could certainly see discussing thing accounting for our 1.0 release, and if so, I'd love your thoughts on the benefits you see there and any other breaking/major changes you don't see in #865 or https://github.com/adobe/spectrum-web-components/projects/1 that you think we should account for in doing so.